docs/developer-docs/daos/nns/neuron-management.mdx

```diff
@@ -315,14 +398,100 @@ The second argument `followees` defines a list of neuron IDs whose decision will
 _Required permissions:_
 Voting and setting following can be done by a neuron's controller or a neuron's hot key.
 
-You can also use the [ICP Rosetta API](/docs/current/developer-docs/defi/rosetta/icp_rosetta/construction_api/voting/) for this workflow.
+### Handling a neuron's maturity
+Recall from the [Neuron overview](/docs/current/developer-docs/daos/nns/neuron-overview) that when a neuron votes it gets rewards in the form of maturity.
+The maturity can either be unstaked, meaning that it can be spawned and converted into ICP at any point, or it can be staked,
+meaning that it is viewed as part of the neuron's stake and contributes to the voting power but it is locked until the neuron is dissolved.
+
+**Spawn a neuron**
+
+When a neuron's maturity has risen above a threshold, one can spawn a portion of the maturity. In a first step, this creates a new neuron in a special spawning state containing the spawned maturity. After 7 days, the maturity is replaced with newly minted ICP, taking some [maturity modulation function](https://wiki.internetcomputer.org/wiki/Maturity_modulation#:~:text=The%20maturity%20modulation%20function%20introduces,NNS%20neurons%20and%20SNS%20neurons.) into account. In the end, the new neuron will be dissolved and have these ICP as stake - which can then be disbursed to any account.
+The new neuron can remain controlled by the same principal as the parent neuron, or be assigned to a new principal.
+
+The following command spawns a neuron.
+```
+type Spawn = record {
+  percentage_to_spawn : opt nat32;
+  new_controller : opt principal;
+  nonce : opt nat64;
+};
+```
+Specify by `percentage_to_spawn` the portion of the maturity that should be spawn into a neuron in percentage. This should be a value between 1 and 100 (inclusive).
+If the spawned neurons should have a different controller than the parent neuron (from which you spawn the maturity), you can optionally define a different controlling principal by `new_controller`. Otherwise, the spawned neuron will have the same controller as the parent neuron.
+Similarly to claiming a neuron (see above), you need to specify the new neuron's ledger address and choose a nonce for this in the last argument `nonce`.
+Based on this nonce and the controlling principal, the neuron's address is uniquely defined (see section _Compute the neuron's address_).
+If no nonce is provided, the governance canister chooses a random one.
+
+_Required permissions:_
+Only a neuron's controller can spawn maturity from a neuron.
+
+**Merge maturity**
+
+This method allows a neuron controller to merge the (unstaked) maturity of a neuron into the neuron's stake.
+The caller can choose a percentage of maturity to merge.
+```
+type MergeMaturity = record {
+ percentage_to_merge : nat32
+};
+```
+_Required permissions:_
+Only a neuron's controller can merge maturity.
```

Comment:
I don't quite understand what the suggestion is here @jessiemongeon1. Is this a bug that the code-part is shown below the diff?

Reply:
Yes, I've edited the comment so that it formats as intended. You will need to add an additional ` to above and below the code so it formats properly in the page though.

Reply:
Addressed directly by hand